python/GafferImageUI/OpenImageIOReaderUI.py

```diff
@@ -127,6 +127,18 @@
 			"Documented in ImageReader, where it is exposed to users."
 		],
 
+		"fileValid" : [
+
+			"description",
+			"""
+			Whether or not the files exists and can be read into memory,
+			value calculated per frame if an image sequence.
+			""",
```

Comment:
I think it's probably worth documenting the interaction with `missingFrameMode` - that `fileValid` is false even if `Black` or `Hold` is taking effect for a missing file.

Reply:
have updated documentation